app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,316 @@
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/tv_movie_cast" />
+
+        <ImageView
+            android:id="@+id/iv_cast_image_2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_2"
+            app:layout_constraintEnd_toStartOf="@+id/iv_cast_image_3"
+            app:layout_constraintStart_toEndOf="@+id/iv_cast_image_1"
+            app:layout_constraintTop_toTopOf="@id/iv_cast_image_1" />
+
+        <ImageView
+            android:id="@+id/iv_cast_image_3"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_3"
+            app:layout_constraintEnd_toStartOf="@+id/iv_cast_image_4"
+            app:layout_constraintStart_toEndOf="@+id/iv_cast_image_2"
+            app:layout_constraintTop_toTopOf="@id/iv_cast_image_1" />
+
+        <ImageView
+            android:id="@+id/iv_cast_image_4"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_4"
+            app:layout_constraintEnd_toStartOf="@id/right_guidline"
+            app:layout_constraintStart_toEndOf="@+id/iv_cast_image_3"
+            app:layout_constraintTop_toTopOf="@id/iv_cast_image_1" />
+
+        <TextView
+            android:id="@+id/tv_cast_imagine_text_1"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="6dp"
+            android:text="@string/robert_downey_jr"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toStartOf="@+id/tv_cast_imagine_text_2"
+            app:layout_constraintStart_toStartOf="@id/iv_cast_image_1"
+            app:layout_constraintTop_toBottomOf="@id/iv_cast_image_1" />
+
+        <TextView
+            android:id="@+id/tv_cast_imagine_text_2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:gravity="start"
+            android:text="@string/chris_evans"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toStartOf="@+id/tv_cast_imagine_text_3"
+            app:layout_constraintStart_toEndOf="@+id/tv_cast_imagine_text_1"
```

Comment:
app:layout_constraintStart_toEndOf="@+id/tv_cast_imagine_text_1"
app:layout_constraintStart_toStartOf="@id/iv_cast_image_2"

remember: for all sides u should have maximum one
app:layout_constraintEnd_to for right side
app:layout_constraintStart_to for left side
app:layout_constraintBottom_to for bottom
app:layout_constraintTop_to for top